src/App.js

```diff
@@ -14,11 +16,87 @@ const goodsFromServer = [
   'Garlic',
 ];
 
-const App = () => (
-  <div className="App">
-    <h1>Goods</h1>
-    {goodsFromServer.length}
-  </div>
-);
+class App extends Component {
+  state = {
+    isGoodsLoaded: false,
+    goods: [...goodsFromServer],
+    goods1: [...goodsFromServer],
+    handleValue: 1,
```

Comment:
что за value, как понять?
```suggestion
    minLength: 1,
```